nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java

```diff
@@ -113,31 +113,28 @@ public void setWriteListener(WriteListener writeListener) {
 
     @Test
     public void testSimple() throws Exception {
-        BulkCommand command = createCommand();
-        testCsvExport(command);
+        testCsvExport(createCommand().build());
     }
 
     @Test
     public void testSimpleWithMultipleBuckets() throws Exception {
-        BulkCommand command = createCommand();
+        BulkCommand command = createCommand().build();
         command.setBucketSize(1);
         command.setBatchSize(1);
         testCsvExport(command);
     }
 
     @Test
     public void testSimpleWithFileUnsorted() throws Exception {
-        BulkCommand command = createCommand(false, false);
-        testCsvExport(command);
+        testCsvExport(createCommand(false, false).build());
     }
 
     @Test
     public void testSimpleWithFileZipped() throws Exception {
-        BulkCommand command = createCommand(false, true);
-        testCsvExport(command);
+        testCsvExport(createCommand(false, true).build());
     }
 
-    public void testCsvExport(BulkCommand command) throws Exception {
+    protected void testCsvExport(BulkCommand command) throws Exception {
```

Comment:
Does it make sense to make parameter `BulkCommand.Builder` of this method and do the `build` inside ?
Also `createCommand` is a bit weird regarding its return type.

Reply:
Giving a builder is not possible for testSimpleWithMultipleBuckets()
Renamed the method to createBuilder()